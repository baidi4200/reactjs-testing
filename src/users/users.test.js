import React from "react";
import renderer from "react-test-renderer";
import Users from "./users";

it("Renders when there are no users registered", () => {
  const user = renderer.create(<Users />).toJSON();
  expect(user).toMatchSnapshot();
});

it("Renders when there is one specific user registered", () => {
  const user = renderer.create(<Users items={["obaid"]} />).toJSON();
  expect(user).toMatchSnapshot();
});
it("Renders when all user are registered", () => {
  const user = renderer
    .create(<Users items={["obaid", "Arslan", "zubair", "ammad"]} />)
    .toJSON();
  expect(user).toMatchSnapshot();
});
