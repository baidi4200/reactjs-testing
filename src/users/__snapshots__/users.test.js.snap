// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders when all user are registered 1`] = `
<h1>
  Items not found
</h1>
`;

exports[`Renders when there are no users registered 1`] = `
<h1>
  Items not found
</h1>
`;

exports[`Renders when there is one specific user registered 1`] = `
<h1>
  Items not found
</h1>
`;
